### PR TITLE
EG-4147: merging upgrade information into one file

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.6/node-upgrade-notes.md [hunk 1]
```diff
@@ -310,20 +305,35 @@ Corda 4 requires Java 8u171 or any higher Java 8 patch level. Java 9+ is not cur
...
 You may now do any checks that you wish to perform, read the logs, and so on. When you are ready, use this command at the shell:
 
 `run setFlowsDrainingModeEnabled enabled: false`
 
 Your upgrade is complete.
 
-
 {{< warning >}}
 If upgrading from Corda Enterprise 3.x, please ensure your node has been upgraded to the latest point release of that
 distribution. See [Upgrade a Corda 3.X Enterprise Node](../3.3/node-operations-upgrading.html#upgrading-a-corda-enterprise-node)
```
Comment (blsemo) [inline content/en/docs/corda-enterprise/4.6/node-upgrade-notes.md:339]: This should also warn that you need to upgrade via 4.x <= 4.5 when coming from 3.x -upgrades from 3.x to 4.6 will fail with potentially bad repercussions for the database.

**Review by blsemo — COMMENTED** [review]
Looks fine apart from one missing upgrade warning from 3.x

File: content/en/docs/corda-enterprise/4.6/node-upgrade-notes.md (hunk 1 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.6/node-upgrade-notes.md]: The warning about upgrading to 4.x before 4.6 was already present, but at the start of the document. I've moved the content of this warning to the start of the document and merged it with the other one - it is more prominent that way, and 3.x users are more likely to see it before they start the upgrade.

**Review by blsemo — APPROVED** [review]
Looks good to me

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**